test/crowdsale/PausableCrowdsale.test.js

```diff
@@ -0,0 +1,153 @@
+const shouldFail = require('../helpers/shouldFail');
+const expectEvent = require('../helpers/expectEvent');
+
+const PausableCrowdsale = artifacts.require('PausableCrowdsaleImpl');
+const SimpleToken = artifacts.require('SimpleToken');
+
+require('../helpers/setup');
+
+contract('PausableCrowdsale', function ([owner, wallet, stranger]) {
+  const rate = 1;
+  const value = 1;
+
+  beforeEach('setting up', async function () {
+    this.token = await SimpleToken.new();
+    this.crowdsale = await PausableCrowdsale.new(rate, wallet, this.token.address);
+    await this.token.transfer(this.crowdsale.address, value);
+  });
+
+  describe('pause()', function () {
+    describe('when the sender is the crowdsale owner', function () {
```

Comment:
I tend to use `context` here, and leave `describe` for features or functionality. See `Pausable.test.js`.